From 49fa2ecc554b4997e2a9014d3ce3a148e2a69429 Mon Sep 17 00:00:00 2001
From: G Davey <gdavey@committhis.co.uk>
Date: Tue, 28 Jul 2020 11:46:58 +0100
Subject: [PATCH 1/1] Add <i2c/smbus.h> to vendor_driver_pool.c

We can't really on ONL providing the correct i2c header definitions, and as it's
a file that doesn't relate to any changes made for mion, we have to add it

Signed-off-by: G Davey <gdavey@committhis.co.uk>
---
 .../x86_64_stordis_bf2556x_1t/module/src/vendor_driver_pool.c    | 1 +
 1 file changed, 1 insertion(+)

diff --git a/packages/platforms/stordis/x86-64/bf2556x-1t/onlp/builds/x86_64_stordis_bf2556x_1t/module/src/vendor_driver_pool.c b/packages/platforms/stordis/x86-64/bf2556x-1t/onlp/builds/x86_64_stordis_bf2556x_1t/module/src/vendor_driver_pool.c
index bfa20e99..dd6f2bab 100644
--- a/packages/platforms/stordis/x86-64/bf2556x-1t/onlp/builds/x86_64_stordis_bf2556x_1t/module/src/vendor_driver_pool.c
+++ b/packages/platforms/stordis/x86-64/bf2556x-1t/onlp/builds/x86_64_stordis_bf2556x_1t/module/src/vendor_driver_pool.c
@@ -24,6 +24,7 @@
  *
  ***********************************************************/
 #include <linux/i2c-dev.h>
+#include <i2c/smbus.h>
 
 #include <onlp/onlp.h>
 #include <onlplib/file.h>
-- 
2.17.1
